### PR TITLE
translate README to Chinese

### PR DESCRIPTION
**Pull request by diguage**

### CODE REVIEW

**Comment by mojavelinux** [comment]
Thanks!

Two requests.

1. Can you undo the change to the Debian section in the English guide? Asciidoctor is no longer limited to Debian Sid or Ubuntu Saucy. It's available in all version of Debian and Ubuntu. The French guide probably needs to be corrected.
2. Can you remove the Changelog section from the Chinese translation and just link to the Changelog file. (I'm probably going to remove it from the English version at some point too). Just translate the version last sentence in the document and leave that as the content of the Changelog section.

File: README.adoc [hunk 1]
```diff
@@ -187,7 +191,7 @@ To upgrade the gem, use:
 
 TIP: Your Fedora system may be configured to automatically update packages, in which case no action is required by you to update the gem.
 
-==== apt-get (Debian, Ubuntu, Mint)
+==== apt-get (Debian Sid, Ubuntu Saucy or greater, Mint)
```
Comment (mojavelinux) [inline README.adoc]: This line is correct the way it is. Other docs should be updated to conform.

File: README.adoc [hunk 2]
```diff
@@ -197,7 +201,7 @@ To upgrade the gem, use:
 
  $ sudo apt-get upgrade -y asciidoctor
 
-TIP: Your system may be configured to automatically update deb packages, in which case no action is required by you to update the gem.
+TIP: Your Debian or Ubuntu system may be configured to automatically update packages, in which case no action is required by you to update the gem.
```
Comment (mojavelinux) [inline README.adoc]: :+1:

**Comment by diguage** [comment]
It is OK. Please review again.

**Comment by mojavelinux** [comment]
Thanks for updating!

It seems like you had a merge conflict when your rebased since this branch now includes changes to other files. I don't mind fixing it if you aren't sure how.

**Comment by diguage** [comment]
I'm sorry. I fixed it. Please review again.

**Comment by diguage** [comment]
I checked the commit log. I found some strange things: the commit log is repeated.

@mojavelinux You said there was  a merge conflict. Then I ran the commands as follows:

```bash
zh-cn>   git checkout master

master> git pull 

master> git checkout zh-cn

zh-cn>   git rebase master   # I'm not sure I should use the command.  git merge master is better?

# deal the conflict

zh-cn>   git pull diguage zh-cn

# auto merge

zh-cn>   git push diguage zh-cn

# Pull Request
```

At last I found the commit log was repeated.

**Comment by mojavelinux** [comment]
I can go ahead and fix it. I've gotten myself into some tight spots with git and I'm pretty familiar with how to untangle it.

**Comment by mojavelinux** [comment]
Merged as 008707a04165584756d142ab45ae8f675ad3782c. Thanks!!

**Pull request closed without merging**